package.json

```diff
@@ -28,11 +28,11 @@
     "moment": "^2.17.1",
     "react": "^15.4.1",
     "react-addons-shallow-compare": "^15.4.1",
-    "react-native": "^0.39.2",
+    "react-native": "neerajwahi/react-native#master",
```

Comment:
Would like to move this fork to `zulip/react-native` but don't have permission to fork on the zulip org. @timabbott could you help with this?